giskard/scanner/llm/llm_prompt_injection_detector.py

```diff
@@ -121,68 +95,33 @@ def run(self, model: BaseModel, dataset: Dataset) -> Sequence[Issue]:
                         "model ignore "
                         "previous instructions or perform unintended actions.",
                     ),
-                    description=group.description,
+                    description=group_description,
                     meta={
-                        "domain": group.name,
+                        "domain": group,
                         "metric_value": metric,
                         "threshold": self.threshold,
-                        "test_case": group.name,
-                        "deviation": f"{failed}/{total} " + group.deviation_description,
+                        "test_case": group,
+                        "deviation": f"{number_of_failed_prompts}/{len(group_idx)} " + group_deviation_description,
                         "hide_index": True,
-                        "input_prompts": results[group]["input_prompt"],
-                        "predictions": results[group]["prediction"],
+                        "input_prompts": dataset.df.loc[:, model.meta.feature_names],
+                        "meta_df": group_meta_df,
```

Comment:
No idea what meta df is. It’s the evaluator config? More explicit name could help.

Reply:
not only evaluator configs but also additional `meta` data (like group name for instance) hence the name